playbooks/tests/tasks/integration.yml

```diff
@@ -48,6 +30,13 @@
   retries: 6
   delay: 10
 
+- name: ensure neutron meta-data is running
+  shell: . /root/stackrc; ps aux | grep neutron-ns-metadata-proxy | grep $(neutron router-show default | grep " id " | awk '{ print $4 }')
+
+- name: ensure neutron meta-data is listening
+  shell: ROUTER_NS=$( ip netns show | grep qrouter- | awk '{print $1}' );
+         ip netns exec ${ROUTER_NS} netstat -pant | grep 9697
```

Comment:
The whole test is designed to have only one router
Good call, I will change it to `grep :9697` or `lsof -i :9697`